src/App.svelte

```diff
@@ -9,7 +9,6 @@
   import Peertube from "./elements/peertube/Peertube.wc.svelte";
   import Funkwhale from "./elements/funkwhale/Funkwhale.wc.svelte";
   import Mattermost from "./elements/Mattermost/Mattermost.wc.svelte";
-  // import Mastodon from "./elements/Mastodon/Mastodon.wc.svelte";
```

Comment:
the changes here in this file are not related to this pr, please get it back

Reply:
alright @AhmedHanafy725